packages/shared/src/components/MenuIcon.tsx

```diff
@@ -2,8 +2,10 @@ import React, { ReactElement } from 'react';
 
 export const MenuIcon = ({
   Icon,
+  filled = false,
 }: {
   Icon: React.ElementType;
+  filled?: boolean;
```

Comment:
I think it would be best to utilize our common props for icons, named `IconProps` to lessen the maintenance overhead whenever we need to modify something related to our base Icon component.